xbmc/cores/VideoPlayer/DVDInputStreams/DVDInputStreamFFmpeg.cpp

```diff
@@ -42,26 +48,23 @@ CDVDInputStreamFFmpeg::~CDVDInputStreamFFmpeg()
 
 bool CDVDInputStreamFFmpeg::IsEOF()
 {
-  if(m_aborted)
```

Comment:
leave this as it was please. it's better for debugging because you can set a break point